src/core/Track.cpp

```diff
@@ -3042,3 +3043,30 @@ void TrackView::createTCOView( TrackContentObject * tco )
 	}
 	tco->selectViewOnCreate( false );
 }
+
+
+/*! \brief Show a message box warning the user that this track is about to be closed */
+bool TrackOperationsWidget::warnRemoveTrack()
+{
+    QString messageRemoveTrack = tr("After removing a track, it can not"
+                                    "be recovered. Are you sure you want to remove track \"");
```

Comment:
You need to insert a space after 'not' or it will be written together with 'be' on the line after.
`tr("After removing a track, it can not "`